soccer/gameplay/constants.py

```diff
@@ -29,6 +29,10 @@ class Robot:
     ChipClearance = (
         .1, .6)  # min and max distance a chip will go over another robot
 
+    class Chipper:
+        MinChipRange = 0.3
+        MaxChipRange = 3
```

Comment:
So there's 3 different versions of chip stuff now, `Robot.Chipper.*`, `Robot.ChipClearance` and `OurChpiping.*`. These should be combined

Reply:
Yeah I was just trying to do this quickly to remove a dependency where a play imported these parameters from another play (like they were in the body of another play, arbitrarily assigned there) that got renamed without looking too much. I'll try to clean it up if we are having chipper parameter set fragmentation.